language/stdlib/modules/ValidatorConfig.move

```diff
@@ -100,16 +98,17 @@ module ValidatorConfig {
         assert(exists_config(sender), Errors::not_published(EVALIDATOR_CONFIG));
         (borrow_global_mut<ValidatorConfig>(sender)).operator_account = Option::some(operator_account);
     }
-
     spec fun set_operator {
-        include Roles::AbortsIfNotValidator;
+        include Roles::AbortsIfNotValidator; // Aborts if not validator [B24].
```

Comment:
Make a documentation comment out of this.

Reply:
Done.